### PR TITLE
PM-title bar color changes

### PR DESCRIPTION
**Pull request by pm-dimagi**
Change of the title bar color in the commcare and the launch icon changes

## Feature Flag
https://dimagi.atlassian.net/browse/CCCT-534

## Product Description
https://www.figma.com/design/UWjp1wRNAO0o9q2llwDKMc/CommCare-Cosmetic-Changes-Post-Nov-'22Rebrand?node-id=1322-3886&t=TgPFUaWgJx2XMlmX-1
Screenshot_20241120_032101_CommCare Debug
WhatsApp Image 2024-11-22 at 08 10 47
WhatsApp Image 2024-11-22 at 08 50 40

## PR Checklist

- [x] If I think the PR is high risk, "High Risk" label is set
- [ ] I have confidence that this PR will not introduce a regression for the reasons below
- [x] Do we need to enhance manual QA test coverage ? If yes, "QA Note" label is set correctly
- [x] Does the PR introduce any major changes worth communicating ? If yes, "Release Note" label is set and a "Release Note" is specified in PR description.

### CODE REVIEW

File: app/res/values/themes.xml
Comment (shubham1g5) [inline app/res/values/themes.xml:78]: Are these values coming from figma as well ? I would think we should keep the style same as `TitleTextStyle` as thats standard Android sizes and font styles.

File: app/res/values/themes.xml
Comment (shubham1g5) [inline app/res/values/themes.xml:78]: Do we not need this on pre O devices ?

File: app/res/values/themes.xml
Comment (shubham1g5) [inline app/res/values/themes.xml:66]: we should delete ther resources we are no longer using.

File: app/res/drawable/logo.xml
Comment (shubham1g5) [inline app/res/drawable/logo.xml:1]: nit: Rename to `commcare_logo.xml`

File: app/res/drawable/logo_spacing.xml
Comment (shubham1g5) [inline app/res/drawable/logo_spacing.xml:6]: is this intentional to not have equal left and right values like the earlier `actionbar_modern_spacing` ?

File: app/res/drawable/logo_spacing.xml
Comment (shubham1g5) [inline app/res/drawable/logo_spacing.xml:1]: nit: rename to `commcare_actionbar_logo_spacing`

**Review by shubham1g5 — CHANGES_REQUESTED** [review]

File: app/res/values/themes.xml
Comment (pm-dimagi) [reply app/res/values/themes.xml:78]: we need it on all but weight 400 is available above this only otherwise we have only bold option

File: app/res/drawable/logo_spacing.xml
Comment (pm-dimagi) [reply app/res/drawable/logo_spacing.xml:6]: yess because as per figma its 16dp in right and by default it takes 16 in left

File: app/res/values/themes.xml
Comment (pm-dimagi) [reply app/res/values/themes.xml:77]: Yes we need it but font weight 400 doses not support below that , we have to use bold or have to get ttf file

File: app/res/drawable/logo_spacing.xml
Comment (pm-dimagi) [reply app/res/drawable/logo_spacing.xml]: left is predefined 16dp so no need to add that
Right is earlier 8dp but as per figma it should be 16dp

File: app/res/values/themes.xml
Comment (shubham1g5) [reply app/res/values/themes.xml:77]: can we use bold instead then to be consistent across devices ?

**Review by shubham1g5 — CHANGES_REQUESTED** [review]
Incomplete PR checklist.

**Review by shubham1g5 — APPROVED** [review]
Approved, though we should not merge until the base PR is merged into `master`

**Pull request merged**
